front/src/components/member/MemberSidebar.vue

```diff
@@ -83,6 +84,18 @@ export default {
     isGuest() {
       return this.$store.getters["member/isGuest"];
     },
+    shortNickname() {
```

Comment:
```suggestion
    shortenedNickname() {
```
```suggestion
    shortNicknameForSidebar() {
```
둘 중 하나로 바꾸는게 어떨까요? 처음에 변수명만 봤을 때 이해가 한번에 안가네요😢 
오구가 제안하신대로 `MAX_NICKNAME_LENGTH` 와 함께 적용하면 좋을 것 같아요